pkg/plugin/loader/loader.go

```diff
@@ -73,7 +75,12 @@ func LoadAllPlugins(namespace, sonobuoyImage, imagePullPolicy string, searchPath
 		pluginDefinitions = append(pluginDefinitions, pluginDefinition)
 	}
 
-	pluginDefinitions = filterPluginDef(pluginDefinitions, selections)
+	// Run every plugin that is loaded unless specifically told otherwise.
+	// This is kept only for some amount of backwards compatibility. We should now only
+	// be loading the plugins we want and may be able to remove this logic soon.
+	if len(selections) > 0 {
```

Comment:
Functional change; again I should consider whether or not I want to differentiate between nil and empty here. That way I can support nil -> run all implicitly vs empty -> I really dont want to run any plugins.